README.md

```diff
@@ -10,3 +17,155 @@ Heat transfer code utilizing a nondimensionalized semi-analytic solution to movi
 ## License
 
 3dThesis is distributed under an [open source 3-clause BSD license](LICENSE).
+
+## Inputs
+This section shows how to create a custom simulation with all tunable parameters.
+If a necessary keyword is not included, often the program will default to values found in the test case. Note when editing files, it is important to begin each group with a brace `{`, as well as close each group `}` as in the example files.
+
+When run from the command line, an input file is optional (e.g. `./3DThesis ParamInput.txt`), defaulting to `TestInputs/ParamInput.txt`. If `Name` is set to `TestSim`, then the data for the simulation can be found at `Data/TestSim`. All the files under `Simulation` are necessary, the rest are optional.
```

Comment:
Did this change? I thought data was output to TestSim/Data.

Reply:
It changed TO TestSim/Data. Shows how old the readme is.